packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -22,6 +23,7 @@ class WidgetFlutterBinding extends FlutterBinding {
     assert(FlutterBinding.instance is WidgetFlutterBinding);
   }
 
+  /// The one static instance of this class.
```

Comment:
Only valid after `ensureInitialized()` (or the WidgetFlutterBinding constructor) has been called. If another FlutterBinding subclass is instantiated before this one (e.g. bindings from other frameworks based on the Flutter "rendering" library), then WidgetFlutterBinding.instance will not be valid (and will throw in checked mode).